chart/ci/manifests/secrets.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+stringData:
+  username: xnatuser
+  password: xnatpass
+  dbname: xnatdb
+kind: Secret
+metadata:
+  name: pg-user-secret
+  namespace: xnat-core
+type: kubernetes.io/basic-auth
+---
+apiVersion: v1
+stringData:
+  username: xnatuser
+  password: xnatpass
+  dbname: xnatdb
+kind: Secret
+metadata:
+  name: xnat-core-postgresql-app
+  namespace: xnat-core
+type: kubernetes.io/basic-auth
```

Comment:
I needed to duplicate the secret for testing as I was running into this issue: https://github.com/cloudnative-pg/charts/issues/501